auto_backup/models/db_backup.py

```diff
@@ -232,6 +232,7 @@ def schedule_backup(self):
                     # Navigate in to the correct folder.
                     sftp.chdir(pathToWriteTo)
 
+                    _logger.info("Checking expired files")
```

Comment:
I'm not sure if this has much value honestly.

Reply:
I added it when I was debugging where the process failed and why we had accumulating SFTP connections. IMO it's useful in certain scenarios to log execution of different functions of the code and doesn't add much unnecessary logging, but it's fine either way.

Reply:
Hmm, I'm fine with it if you convert it to a debug 😉

Reply:
Sure, done :)